targets/TARGET_ARM_FM/TARGET_FVP_MPS2/TARGET_FVP_MPS2_M3/device/TOOLCHAIN_GCC_ARM/MPS2.ld

```diff
@@ -62,11 +66,10 @@ MEMORY
  */
 ENTRY(Reset_Handler)
 
-HEAP_SIZE  = 0x4000;
-STACK_SIZE = 0x1000;
+STACK_SIZE = 0x400;
```

Comment:
Stack size is hard-coded to `1KB` not only for `IAR`, but also for `GCC` and `ARM` compilers. `1KB` is the stack size required by `MBED 5`. And I see that there is no distinction for `MBED 2` builds (which requires 4KB stack). 
I see that `FVP_MPS2` target has `release_version` `5` only.  Probably `MBED 2` builds are are not provided for Fast Model and that is why there is no distinction for stack required by `MBED 2`. Is it correct?

Reply:
Yes, hardcoded for 1KB is because Fastmodels only support mbed OS 5 at moment.
When mbed os 2 support is added, the linker scripts need to be adjusted based on the presence of `MBED_CONF_RTOS_PRESENT` macro